students/JimJenkins/session01/break_me.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+'''
+Author: Jim Jenkins (dvlupr)
+Date: 10/13/2018
+
+Each function, when called, should cause an exception to happen
+Each function should result in one of the four most common exceptions you’ll find.
+For review: NameError, TypeError, SyntaxError, AttributeError
+
+'''
+# functions that trigger each error
+def fn_nameerror():
+    print nameerror
```

Comment:
Missing the parentheses and quotes on this one - I'm getting a SyntaxError

Reply:
NOTE: ifthis code runs, then you are running it in Python2 -- you really do want to figure out how to get Python3 running -- they aren't all that different, but we do use Python3 only features in this class.